HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -129,6 +129,13 @@ def customiseFor2018Input(process):
 
     return process
 
+def customizeForCSCRecHitDProducer(process):
```

Comment:
Please rename as 
```suggestion
def customizeFor33453(process):
```
so that @Martin-Grunewald can retrieve and remove this temporary customization more easily once this PR is merged